### PR TITLE
invalidate stale commits for PRs too

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: .github/rm-invalid-lowest-lock-files.php [hunk 1]
```diff
@@ -104,8 +104,7 @@ function getContentHash(array $composerJson)
     }
 }
 
-if (!$referencedCommits || (isset($_SERVER['TRAVIS_PULL_REQUEST']) && 'false' !== $_SERVER['TRAVIS_PULL_REQUEST'])) {
-    // cached commits cannot be stale for PRs
```
Comment (xabbuh) [inline .github/rm-invalid-lowest-lock-files.php]: it seems they can: https://travis-ci.org/symfony/symfony/jobs/438809475

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Comment by xabbuh** [comment]
@nicolas-grekas Can you check this change nonetheless? It makes build pass again, but I am not fully sure that I did not break any legitimate use case (i.e. maybe we now invalidate too often and thus slowing builds down again).

File: .github/rm-invalid-lowest-lock-files.php (hunk 1 above)
Comment (nicolas-grekas) [reply .github/rm-invalid-lowest-lock-files.php]: That's good: now than we use flex as an accelerator, this is not really slow anymore.

Comment (xabbuh) [reply .github/rm-invalid-lowest-lock-files.php]: thanks

**Pull request merged**